.github/workflows/tests.yml

```
@@ -56,6 +56,22 @@ jobs:
           unzip -o dcm2niix_lnx.zip
           sudo install dcm2nii* /usr/bin/
 
+      - name: SpinalCord Toolbox v5.0.1 installation macOS
+        if: contains(matrix.os, 'macos')
+        run: |
+          mkdir spinalcordtoolbox-5.0.1
+          curl -o spinalcordtoolbox-5.0.1/spinalcordtoolbox-5.0.1 -JL https://github.com/neuropoly/spinalcordtoolbox/archive/5.0.1.zip
+          cd spinalcordtoolbox-5.0.1
+          yes | ./install_sct
+
+      - name: SpinalCord Toolbox v5.0.1 installation Ubuntu
+        if: contains(matrix.os, 'ubuntu')
+        run: |
+          curl -JLO https://github.com/neuropoly/spinalcordtoolbox/archive/5.0.1.zip
+          unzip -o spinalcordtoolbox-5.0.1.zip
+          cd spinalcordtoolbox-5.0.1
+          yes | ./install_sct
+
```

If these are the same zip src being installed for both macOS and Ubuntu, you can collapse these into a single Workflow step (remove the `if: contains()` statement:
```suggestion
      - name: SpinalCord Toolbox v5.0.1 installation
        run: |
          mkdir spinalcordtoolbox-5.0.1
          curl -o spinalcordtoolbox-5.0.1/spinalcordtoolbox-5.0.1 -JL https://github.com/neuropoly/spinalcordtoolbox/archive/5.0.1.zip
          cd spinalcordtoolbox-5.0.1
          yes | ./install_sct
```

---

I tried it yesterday, it worked for the Ubuntu tests but the installation of SCT on macOS was just cancelled after running for 6 hours...

---

Hmm yeah, looks like the recent commit is also taking a long time to install SCT 🤔

---

Yes, I think it will give the same result again...

---

I interrupted the build to get a peek at the logs... looks like the macOS build never reaches [this line](https://github.com/shimming-toolbox/shimming-toolbox/pull/184/checks?check_run_id=1507641746#step:8:2128) that's reached in the Ubuntu build.

---

Exactly the same line that wasn't building when I was implementing SCT the way you proposed!

---

It's interesting that macOS doesn't error out where Ubuntu does with the `yes: standard output: Broken pipe`. This hints that either `yes` or piping works differently on macOS. I wonder if it's related to this [issue](https://stackoverflow.com/a/20624315/6310633).

---

We've solved this and merged a fix to `master` today: https://github.com/neuropoly/spinalcordtoolbox/commit/d64af23b69dbc396322cdc963a90a899b73736e0

That SO thread is interesting. I'm going to forward that to https://stackoverflow.com/questions/20573282/hudson-yes-standard-output-broken-pipe/20624315#20624315

If you install from SCT's `master` it should be good to go.